### ISSUE TITLE
printing issue

### ISSUE THREAD

**Issue by cedricvanrompay** [OCamlPro/tryocaml#154]
I think there is a problem with printing.
Here is what I got:

```
# print_int 3;;
- : unit = <unknown constructor>
# print_string "Hello";;
- : unit = <unknown constructor>
# print_string "x = ";;
- : unit = <unknown constructor>
# ;
File "", line 1, characters 21-22:
Error: Syntax error
# print_string "x = "; print_int !x; print_newline ();;
- : unit = ()
# Printf.printf "x = %d. Bye %s\n" !x "John";;
- : unit = ()
# Printf.printf "Bye %s\n" "John";;
- : unit = ()
```

**Comment by sfcgeorge** [comment]
Yep I get that too. And a lot of the tutorial involves printing so it's kinda useless when all you see is `- : unit = ()`.

**Comment by Daniel-Hug** [comment]
I found out you have to open the browser's console to see the printed values. In chrome that's <kbd>f12</kbd> (open devtools), then <kbd>ESC</kbd> (open console drawer). This is not obvious at all from the tutorial though. :(

**Comment by pthedinger** [comment]
It's probably worth noting that in Chrome I'm seeing that
  print_int 3
doesn't come out to the console straight away either - it is buffered. So you have to wait until there is a newline to be able to see it. Just having the console open won't be enough

**Comment by markbao** [comment]
+1, still not fixed as of today

**Comment by hhugo** [comment]
https://github.com/OCamlPro/tryocaml/issues/146

**Comment by markbao** [comment]
@hhugo Nice, thanks. Does that mean that someone just needs to make a PR to update the js-of-ocaml dependency? I would help but I'm not very familiar with this.

**Comment by hhugo** [comment]
I never managed to build tryocaml myself https://github.com/OCamlPro/tryocaml/issues/135.
The compilation process was non standard and rely on compiler patches (https://github.com/OCamlPro/tryocaml/tree/master/toplevellib) with no support for latest version of ocaml.
The situation may have change, but I don't see anything recent in the commit history.
Also, the website is using a 2 years old version of js_of_ocaml.

An up to date ocaml toplevel is available here : 
http://ocsigen.org/js_of_ocaml/dev/files/toplevel/index.html

**Comment by cagdasbozman** [comment]
We actually are working on a new interface which will be online soon.

**Comment by luiscruz** [comment]
+1

**Comment by zzz6519003** [comment]
@Daniel-Hug  thx!

**Comment by chindo** [comment]
escape problems.

While the "new interface" does not arrive, you can use:

`print_int\ 2`

**Comment by ericjmorey** [comment]
Problem persists:
https://github.com/OCamlPro/tryocaml/issues/146#issuecomment-553676417
